packages/ds-medicare-gov/src/styles/components/_Tooltip.scss

```diff
@@ -6,14 +6,14 @@
 .ds-c-tooltip__trigger-link {
   color: var(--tooltip-trigger__color);
   cursor: default;
-  text-decoration-color: var(--color-primary);
+  text-decoration-color: var(--tooltip-trigger__color);
   text-decoration-style: dashed;
   text-decoration-thickness: var(--link__text-decoration-thickness);
   text-underline-offset: var(--link__text-decoration-offset);
 
   &:hover {
     color: var(--color-black);
-    text-decoration-color: var(--color-primary-darker);
+    text-decoration-color: var(--color-black);
```

Comment:
Is this a temporary decision? Should it be flagged to revisit later?

Reply:
Also, I found through testing that you've overridden the hover color for medicare but not for any of the other themes. Is that intentional?

Reply:
No that was a whoopsie. Thank you for finding that. 

I need to document in the ticket the intent behind these changes and that they will likely be reviewed and possibly changed as we embark on the tooltip discovery work. I think at this moment I don't know if these changes are temporary in such a way that I know for a fact they will be removed and when. So given that, I think documentation in the ticket should be sufficient, but also happy to add a comment to the SCSS file.

Reply:
👍 It's up to you how you want to document it. I just wanted you to be able to remember it.